app/posts/[identifier]/page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { getMumblePostAction } from '@/actions/getMumblePost';
+import { PostCard } from '@/components/post-card/PostCard';
+import { PostEditor } from '@/components/post-editor/PostEditor';
+import PostActionsBar from '@/components/post-actions-bar/PostActionsBar';
+
+export default async function Page({
+  params,
+}: {
+  params: { identifier: string };
```

Comment:
In my PR I used the keyword "id" for the identifier: `app/profiles/[id]/page.tsx`
we should agree on the same word for consistency.

Reply:
i like more identifier. we can flip a coin  😄

Reply:
identifier will be used!